### PR TITLE
change name to Id instead of attributes

### PR DESCRIPTION
**Pull request by e7mac**
Fixing the method name since we had 2 params with the same name in the method name

### CODE REVIEW

File: Leanplum/Api/LPDeviceApi.m [hunk 1]
```diff
@@ -14,9 +14,10 @@
 
 @implementation LPDeviceApi
 
-+ (void) setDeviceAttributes:(NSString *)deviceId withDeviceAttributes:(NSDictionary *)attributes
-                     success:(void (^)(void))success
-                     failure:(void (^)(NSError *error))failure {
++ (void) setDeviceId:(NSString *)deviceId
+withDeviceAttributes:(NSDictionary *)attributes
+             success:(void (^)(void))success
+             failure:(void (^)(NSError *error))failure {
```
Comment (gg4race) [inline Leanplum/Api/LPDeviceApi.m]: why is this change necessary? In old SDK the api is setDeviceAttributes not setDeviceId

File: Leanplum/Api/LPUserApi.h [hunk 2]
```diff
@@ -10,10 +10,10 @@
 
 @interface LPUserApi : NSObject
 
-+ (void) setUserAttributes:(NSString *)userId
-        withUserAttributes:(NSDictionary *)attributes
-                   success:(void (^)(void))success
-                   failure:(void (^)(NSError *error))failure;
++ (void) setUserId:(NSString *)userId
```
Comment (gg4race) [inline Leanplum/Api/LPUserApi.h]: same here

File: Leanplum/Api/LPDeviceApi.m (hunk 1 above)
Comment (hrishileanplum) [reply Leanplum/Api/LPDeviceApi.m]: i think @gg4race , the change is easier to read, cos we do have "withDeviceAttributes" as a param name as well.
